packages/client/mutations/CreateJiraIssueMutation.ts

```diff
@@ -59,6 +59,7 @@ const CreateJiraIssueMutation = (
       const userId = task.getValue('userId')
       const teamId = task.getValue('teamId')
       const integration = task.getLinkedRecord('integration')
+      if (!userId) return
       const user = store.get(userId)
       if (!user || !integration) return
```

Comment:
It looks Jira integrations require a `user` so I'm removing the integration button from the unassigned tasks